go.mod

```diff
@@ -24,10 +24,12 @@ require (
 	github.com/Azure/azure-sdk-for-go v30.0.0+incompatible
 	github.com/Azure/azure-service-bus-go v0.8.0
 	github.com/Azure/azure-storage-blob-go v0.6.0
-	github.com/Azure/go-autorest v12.0.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.5.0
```

Comment:
Please revert all of the `go.mod` and `go.sum` changes, they shouldn't be needed.

Reply:
The changes are to be able to reference `azure.PublicCloud.ResourceIdentifiers.Storage`. Will revert if you want

Reply:
Yes, please revert; we are doing a release soon and don't want to upgrade dependencies right now. Thanks!